vsphere/tests/test_check.py

```diff
@@ -431,3 +432,34 @@ def test_specs_start_time(aggregator, dd_run_check, historical_instance):
     assert len(start_times) != 0
     for start_time in start_times:
         assert start_time == (mock_time - dt.timedelta(hours=2))
+
+
+@pytest.mark.parametrize(
+    'test_timeout, expected_result',
+    [
+        (5, False),
+        (10, False),
+        (20, True),
+        ('5', False),
+        ('20', True),
+    ],
+)
+@pytest.mark.usefixtures('mock_type', 'mock_threadpool', 'mock_api', 'mock_rest_api')
```

Comment:
Just curious what are these fixtures used for?

Reply:
Ah sorry. There's only 2 we need in here. `mock_type` and `mock_api`. the former changes the return of `type(MagicMock(spec=int)) = MagicMock` to `int` otherwise the check will throw an exception. The latter for the api connection.